fs/copy.ts

```diff
@@ -137,7 +137,7 @@ async function copySymLink(
 ) {
   await ensureValidCopy(src, dest, options);
   const originSrcFilePath = await Deno.readLink(src);
```

Comment:
Using `Deno.realPath` should return the absolute path of the symlink target. Additional explanation for suggestion in PR #6236:
```suggestion
  const originSrcFilePath = await Deno.realPath(src);
```

Reply:
I think changing this to `realPath` causes another problem. I think we should keep this work for relative symlinks.